microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -250,49 +250,39 @@ def _extract_test_results(
         # { "type": "test", "event": "ok", "name": "integration::test_vfio" }
         lines = output.split("\n")
         for line in lines:
-            matches = re.findall(r"{.*}", line)
-            if not matches:
+            if '"type": "test"' not in line:
```

Comment:
Why do we need this? This is not a reliable check. In the future, cargo might format the JSON without the space i.e. `"type":"test"`.

Reply:
ok, I'll remove this check. This check isn't a necessity.